src/main/java/duke/exception/MissingArgumentException.java

```diff
@@ -0,0 +1,10 @@
+package duke.exception;
+
+/**
+ * DukeException when an argument is missing.
+ */
+public class MissingArgumentException extends DukeException {
+    public MissingArgumentException(String description) {
+        super(description);
+    }
+}
```

Comment:
I like how you wrote more exception classes for `DukeException`, makes exceptions a lot clearer in the code 👍